### PR TITLE
Refactor/1forge types

### PR DESCRIPTION
**Pull request by karen-stepanyan**
## Changes

- Adds forex endpoint ResponseSchema

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/sources/ncfx/src/endpoint/forex.ts [hunk 1]
```diff
@@ -4,8 +4,18 @@ import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
 export const supportedEndpoints = ['forex']
 
 export const inputParameters: InputParameters = {
-  base: ['base', 'from', 'coin'],
-  quote: ['quote', 'to', 'market'],
+  base: {
+    aliases: ['from', 'coin'],
+    description: 'The symbol of the currency to query',
+    options: ['BTC', 'ETH', 'USD'],
```
Comment (justinkaseman) [inline packages/sources/ncfx/src/endpoint/forex.ts:10]: These aren't forex symbols.

File: packages/sources/ncfx/src/endpoint/crypto.ts [hunk 2]
```diff
@@ -4,8 +4,18 @@ import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
...
+  base: {
+    aliases: ['from', 'coin'],
+    description: 'The symbol of the currency to query',
+    options: ['BTC', 'ETH', 'USD'],
+    required: true,
+  },
+  quote: {
+    aliases: ['to', 'market'],
+    description: 'The symbol of the currency to convert to',
+    options: ['BTC', 'ETH', 'USD'],
```
Comment (justinkaseman) [inline packages/sources/ncfx/src/endpoint/crypto.ts:16]: This endpoint supports many more symbols than this. We should either pull an accurate list from the data provider or leave this off.

CC: @austinborn

Comment (justinkaseman) [reply packages/sources/ncfx/src/endpoint/crypto.ts:16]: Actually, this is just used for the README huh? Not actually enforced?

Comment (karen-stepanyan) [reply packages/sources/ncfx/src/endpoint/crypto.ts]: > Actually, this is just used for the README huh? Not actually enforced?

no, enforced. if there is 'options', then values should match

Comment (austinborn) [reply packages/sources/ncfx/src/endpoint/crypto.ts]: Yeah it's enforced. If these are only a sample of options, then it should go in the description section

**Review by justinkaseman — APPROVED** [review]

**Pull request merged**